### ISSUE TITLE
Evaluate existing RDF delta/diff formats

### ISSUE THREAD

**Issue by elf-pavlik** [solid/notifications#157]
Once we merge #61 we should evaluate suggested technologies:

* [ ] https://www.w3.org/2001/sw/wiki/How_to_diff_RDF
* [ ] https://afs.github.io/rdf-delta/
* [ ] https://github.com/solid/specification/issues/332

**Comment by joepio** [comment]
I've discussed even more alternatives in #161

**Comment by csarven** [comment]
One of the topics of 2023-02-22 CG meeting was to move this issue to solid/specification: https://github.com/solid/specification/blob/2cb1a31a47a6f6316918e67bbbd5b881b50fb23c/meetings/2023-02-22.md#evaluate-existing-rdf-deltadiff-formats but we didn't get to pick it up then. We can pick up both issues together in this week's meeting (2023-03-01). We don't have to close this issue just yet in case there are unique cases for notifications.

**Comment by CxRes** [comment]
**FYI [RFC3229](https://www.rfc-editor.org/rfc/rfc3229.html)**

**Comment by elf-pavlik** [comment]
https://www.rfc-editor.org/rfc/rfc3229.html#page-22

>  Our proposal does not recommend any specific algorithm or format, but
   rather encourages client and server implementors to choose the most
   appropriate one(s).

**Comment by CxRes** [comment]
True, but the way you provide diffs according to this standard is different from directly providing a media type for the diff. Instead, you provide media-type for content and an algo for diffing. Also, this technique would be general for both RDF and non-RDF resources. Thats what I wanted to bring to everyone's attention.
